tests/model_serving/model_server/keda/test_isvc_keda_scaling.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+from simple_logger.logger import get_logger
+from typing import Any, Generator
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+from utilities.constants import KServeDeploymentType
+from tests.model_serving.model_server.utils import (
+    verify_keda_scaledobject,
+    run_vllm_concurrent_load,
+    run_ovms_concurrent_load,
+)
+from utilities.infra import get_pods_by_isvc_label
+from tests.model_serving.model_runtime.vllm.constant import (
+    COMPLETION_QUERY,
+    CHAT_QUERY,
+    BASE_RAW_DEPLOYMENT_CONFIG,
+)
+from utilities.constants import ModelFormat, ModelVersion, RunTimeConfigs
+
+LOGGER = get_logger(name=__name__)
+
+
+SERVING_ARGUMENT: list[str] = [
```

Comment:
used once, please use where needed

Reply:
I didn't understand, the serving argument is used to construct the `BASE_RAW_DEPLOYMENT_CONFIG` which is used by the pytest params for creating the isvc